guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

```diff
@@ -5,18 +5,10 @@
 
 We appreciate your input on our documentation. Please let us know how we could make it better.
 
-* For simple comments on specific passages:
-+
-. Ensure you are viewing the documentation in the _Multi-page HTML_ format.
-+
-In addition, ensure you see the *Feedback* button in the upper right corner of the document.
-. Use your mouse cursor to highlight the part of text that you want to comment on.
-. Click the *Add Feedback* pop-up that appears below the highlighted text.
-. Follow the displayed instructions.
+You can submit feedback through Bugzilla.
+To create a new ticket:
 
-* For submitting feedback via Bugzilla, create a new ticket:
-+
 . Go to the link:https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Satellite[Bugzilla] website.
-. As the Component, use *Documentation*.
-. Fill in the *Description* field with your suggestion for improvement. Include a link to the relevant part(s) of documentation.
+. As the *Component*, use `Documentation`.
+. In the *Description* field, enter your suggestion for improvement. Include a link to the relevant part(s) of documentation.
```

Comment:
```suggestion
. In the *Description* field, enter your suggestion for improvement. Include a link to the relevant parts of documentation.
```
Alternatively, you could write "part or parts", but this ^ is what RHEL docs use.

Reply:
Also, can we follow one sentence per line here?